### PR TITLE
added thirdweb integration

### PR DESCRIPTION
**Pull request by shrimalmadhur**
- Call thirbweb listing API
- render into the main component

### CODE REVIEW

File: pages/index.tsx [hunk 1]
```diff
@@ -75,6 +76,7 @@ const Home: NextPage<Props> = ({ fallback }) => {
 
   return (
     <Layout navbar={{}}>
+      <ThirdWebList data={{collectionId: '0x6a5c9776d259f76aa445fc4FCC0cF14843f3DE74'}}></ThirdWebList>
```
Comment (jasonyjong) [inline pages/index.tsx:79]: I would just directly pass in the collectionId:

```
<ThirdWebList collectionId="id" />

# OR 

const collectionId = "uuid";
<ThirdWebList collectionId={collectionId} />
```

Comment (shrimalmadhur) [reply pages/index.tsx:79]: I think the reason I passed like this is the component expects it in the way. I just reused they format other component used. Is there any best practices I should consider @jasonyjong ?
```
type Props = {
    data: {
        collectionId: string | undefined
    }
}
const ThirdWebList: FC<Props> = ({
    data
  }) => {
```

Comment (jasonyjong) [reply pages/index.tsx:79]: Oh interesting -- I took a look at the other Reservoir components and makes sense now. I'm not used to having a generic `data` arg but def okay w/ it to keep things consistent!

**Review by jasonyjong — APPROVED** [review]

File: lib/fetch/fetch.ts [hunk 2]
```diff
@@ -44,3 +47,15 @@ export async function getCollection(
 
   setCollection(json)
 }
+
+export async function getThirdWebCollection(
+  collectionId: string | undefined,
+  setCollection: Dispatch<any>
+) {
+  const marketplace = sdk.getMarketplace("0x6a5c9776d259f76aa445fc4FCC0cF14843f3DE74");
```
Comment (jasonyjong) [inline lib/fetch/fetch.ts:55]: oh i guess we still need to make this generic lol

Comment (shrimalmadhur) [reply lib/fetch/fetch.ts]: ya - made it more generic.

**Pull request merged**